### PR TITLE
[Angular] Tests: delete 'describe' useless

### PR DESCRIPTION
**Pull request by qmonmert**
Some **describe** are useless

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Review by mshima — APPROVED** [review]
Great.

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs [hunk 1]
```diff
@@ -38,289 +38,287 @@ import { I<%= entityAngularName %><% if (!readOnly) { %>, <%= entityAngularName
...
-    <%_ if (field.fieldTypeLocalDate) { _%>
-                    <%= field.fieldName %>: currentDate.format(DATE_FORMAT),
-    <%_ } else if (field.fieldTypeTimed) { _%>
-                    <%= field.fieldName %>: currentDate.format(DATE_TIME_FORMAT),
-    <%_ } _%>
+<%_ if (field.fieldTypeLocalDate) { _%>
+                <%= field.fieldName %>: currentDate.format(DATE_FORMAT),
+<%_ } else if (field.fieldTypeTimed) { _%>
+                <%= field.fieldName %>: currentDate.format(DATE_TIME_FORMAT),
+<%_ } _%>
```
Comment (mshima) [inline generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs:97]: EJS ident should be kept

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs [hunk 2]
```diff
@@ -38,289 +38,287 @@ import { I<%= entityAngularName %><% if (!readOnly) { %>, <%= entityAngularName
...
+<%_ } else if (field.fieldTypeCharSequence) { _%>
+            <%= field.fieldName %>: 'AAAAAAA',
+<%_ } else if (field.fieldTypeTemporal) { _%>
+            <%= field.fieldName %>: currentDate,
+<%_ } else if (field.fieldTypeBinary && !field.blobContentTypeText) { _%>
+            <%= field.fieldName %>ContentType: 'image/png',
+            <%= field.fieldName %>: 'AAAAAAA',
+<%_ } else { /* (fieldType === 'byte[]' || fieldType === 'ByteBuffer') && fieldTypeBlobContent === 'any' || (fieldType === 'byte[]' || fieldType === 'ByteBuffer') && fieldTypeBlobContent === 'image' || fieldType === 'LocalDate' */ _%>
+            <%= field.fieldName %>: 'AAAAAAA',
+<%_ } _%>
```
Comment (mshima) [inline generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs:84]: EJS ident should be kept.

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs [hunk 3]
```diff
@@ -38,289 +38,287 @@ import { I<%= entityAngularName %><% if (!readOnly) { %>, <%= entityAngularName
...
-    <%_ } else { _%>
-                    id: 0,
-    <%_ } _%>
+        it('should create a <%= entityAngularName %>', () => {
+            const returnedFromService = Object.assign({
+<%_ if (tsKeyType === 'string') { _%>
+                id: 'ID',
+<%_ } else { _%>
+                id: 0,
+<%_ } _%>
```
Comment (mshima) [inline generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs:115]: EJS ident.

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs [hunk 4]
```diff
@@ -38,289 +38,287 @@ import { I<%= entityAngularName %><% if (!readOnly) { %>, <%= entityAngularName
...
-    <%_ if (field.fieldTypeLocalDate) { _%>
-                    <%= field.fieldName %>: currentDate.format(DATE_FORMAT),
-    <%_ } else if (field.fieldTypeTimed) { _%>
-                    <%= field.fieldName %>: currentDate.format(DATE_TIME_FORMAT),
-    <%_ } _%>
+<%_ if (field.fieldTypeLocalDate) { _%>
+                <%= field.fieldName %>: currentDate.format(DATE_FORMAT),
+<%_ } else if (field.fieldTypeTimed) { _%>
+                <%= field.fieldName %>: currentDate.format(DATE_TIME_FORMAT),
+<%_ } _%>
```
Comment (mshima) [inline generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs:121]: EJS ident.

**Review by mshima — CHANGES_REQUESTED** [review]
Oops, at entity service, the EJS ident should be reverted.

**Comment by qmonmert** [comment]
@mshima done

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs [hunk 5]
```diff
@@ -38,289 +38,287 @@ import { I<%= entityAngularName %><% if (!readOnly) { %>, <%= entityAngularName
...
 <%_ if (fieldsContainDate) { _%>
-            currentDate = dayjs();
+        currentDate = dayjs();
 <%_ } _%>
 
-            elemDefault = {
+        elemDefault = {
 <%_ fields.forEach((field) => {
-    const fieldType = field.fieldType;
+const fieldType = field.fieldType;
```
Comment (mshima) [inline generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs:65]: ejs ident

**Review by mshima — COMMENTED** [review]
A few more places to adjust ejs ident.

**Comment by qmonmert** [comment]
@mshima done :)

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs [hunk 6]
```diff
@@ -38,289 +38,287 @@ import { I<%= entityAngularName %><% if (!readOnly) { %>, <%= entityAngularName
...
-                const req = httpMock.expectOne({ method: 'PATCH' });
-                req.flush(returnedFromService);
-                expect(expectedResult).toMatchObject(expected);
-            });
-        <%_ } _%>
+            const req = httpMock.expectOne({ method: 'PATCH' });
+            req.flush(returnedFromService);
+            expect(expectedResult).toMatchObject(expected);
+        });
+    <%_ } _%>
```
Comment (mshima) [inline generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs:216]:
```suggestion
<%_ } _%>
```

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs [hunk 7]
```diff
@@ -38,289 +38,287 @@ import { I<%= entityAngularName %><% if (!readOnly) { %>, <%= entityAngularName
...
+
+            service.query().subscribe(resp => expectedResult = resp.body);
+
+            const req = httpMock.expectOne({ method: 'GET' });
+            req.flush([returnedFromService]);
+            httpMock.verify();
+            expect(expectedResult).toContainEqual(expected);
+        });
+
+    <%_ if (!readOnly) { _%>
```
Comment (mshima) [inline generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs:255]:
```suggestion
<%_ if (!readOnly) { _%>
```

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs [hunk 8]
```diff
@@ -38,289 +38,287 @@ import { I<%= entityAngularName %><% if (!readOnly) { %>, <%= entityAngularName
...
+        it('should delete a <%= entityAngularName %>', () => {
+            service.delete(<%- tsKeyId %>).subscribe(resp => expectedResult = resp.ok);
 
-                service.query().subscribe(resp => expectedResult = resp.body);
+            const req = httpMock.expectOne({ method: 'DELETE' });
+            req.flush({ status: 200 });
+            expect(expectedResult);
+        });
+    <%_ } _%>
```
Comment (mshima) [inline generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.spec.ts.ejs:263]:
```suggestion
<%_ } _%>
```

**Review by mshima — APPROVED** [review]
3 minor adjusts.
And we are ready to merge.

**Pull request merged**